Source/MonoGame.Extended.Tests/AngleTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class AngleTest
+    {
+        private const float DELTA = 0.0000001f;
```

Comment:
@craftworkgames Maybe we could push the .DotSettings files of the various projects as well. I think (but alas, I only THINK that) it's where Resharper keeps the project-specific settings. Would be awesome if everyone owning Resharper would have the same settings automatically...